src/components/GameList/GameListCard.svelte

```diff
@@ -7,22 +7,20 @@
 </script>
 
 <article>
-	<div>
-		<h3>{titleGame}</h3>
+	<a class="hide-underline" href="/spel/{gameSlug}">
+		<h1>{titleGame}</h1>
 		<div class="label-container">
 			<p class="label">{groups}</p>
 			<p class="label">{gameName}</p>
 			<p class="label">{personAmount}</p>
 		</div>
-	</div>
-	<button>
-		<a class="hide-underline" href="/spel/{gameSlug}">></a>
-	</button>
+	</a>
+	<span>></span>
```

Comment:
Dit is nog wel even een dingetje. @iSirThijs wat denk jij dat slim is? Dit gaat over het card item. Ik heb nu de hele card clickable gemaakt, maar de span (knop met `>`) is nu niet clickable. Er zijn verschillende dingen mogelijk.

1. Je maakt van de span een a, zodat deze ook klikbaar is (liever niet)
2. Je doet de span 1 line omhoog, zodat deze in de `<a>` valt. Styling klopt dan niet meer en er zal dus een extra div om de `h1` en andere `div` moeten komen.

![image](https://user-images.githubusercontent.com/34123512/118821236-56387a80-b8b7-11eb-9e88-1d3b9b9a0adf.png)